lib/msal-browser/src/cache/BrowserCacheManager.ts

```diff
@@ -962,7 +962,8 @@ export class BrowserCacheManager extends CacheManager {
         // Ensure we don't overwrite interaction in progress for a different clientId
         const key = `${Constants.CACHE_PREFIX}.${TemporaryCacheKeys.INTERACTION_STATUS_KEY}`;
         if (inProgress) {
-            if (this.getInteractionInProgress()) {
+            // For pairwise broker, ensure same frame embedded app can invoke interaction, as both instances will set this flag
+            if (this.isInteractionInProgress(true)) {
```

Comment:
We may need to think about this some more. This will break scenarios where an app has multiple clientIds that may be invoking interaction, such as Sharepoint's scenarios.

Reply:
Maybe have the embedded frame either not call `setInteractionInProgress` or have it pass in a flag that indicates we shouldn't set it?

Reply:
> We may need to think about this some more. This will break scenarios where an app has multiple clientIds that may be invoking interaction, such as Sharepoint's scenarios.

Break in what way?

Reply:
> Maybe have the embedded frame either not call setInteractionInProgress

This could work, but there be side effects as well.

Reply:
Do we understand what could break here?  Or what the side effects of the alternative approach would be?

Reply:
I think the approach of not setting interaction in progress when the application is embedded is better, shouldn't be an issue. PR: https://github.com/AzureAD/microsoft-authentication-library-for-js/pull/4893/files